besu/src/main/java/org/hyperledger/besu/cli/options/unstable/ChainDataPruningOptions.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+package org.hyperledger.besu.cli.options.unstable;
+
+import org.hyperledger.besu.util.number.PositiveNumber;
+
+import picocli.CommandLine;
+
+public class ChainDataPruningOptions {
+
+  public static final long DEFAULT_CHAIN_DATA_PRUNING_MIN_BLOCKS_RETAINED = 50400;
```

Comment:
Rename to MAX_CHAIN_DATA_PRUNING_MIN_BLOCKS_RETAINED? This not just the default but also the maximum number of blocks that will be pruned.

Reply:
This is actually the minimum value for the blocks to retain. I've also changed it to be much smaller.